src/app/performance-display/performance-display.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "../styles/main";
+
+#title {
+  text-align: center;
+  color: $black;
```

Comment:
Exactly, this is the right way, nice. But you still have to change it in the Strides PR.